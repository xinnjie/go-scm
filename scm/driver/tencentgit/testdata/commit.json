{
  "id": "6269835cafdd89e6d8b8d4a4e3738c79206c1d06",
  "message": "推送新内容",
  "parent_ids": [
    "11a63fa022906668338166ce7f2e7bf35d502285"
  ],
  "authored_date": "2012-06-07T07:38:33+0000",
  "author_name": "git_user1",
  "author_email": "git_user1@tencent.com",
  "committed_date": "2012-06-07T07:38:33+0000",
  "committer_name": "git_user1",
  "committer_email": "git_user1@tencent.com",
  "title": "推送新内容",
  "created_at": "2012-06-07T07:38:33+0000",
  "short_id": "6269835c"
}