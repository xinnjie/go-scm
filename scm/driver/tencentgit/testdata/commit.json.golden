{
  "Sha": "6269835cafdd89e6d8b8d4a4e3738c79206c1d06",
  "Message": "推送新内容",
  "Author": {
    "Name": "git_user1",
    "Email": "git_user1@tencent.com",
    "Date": "2012-06-07T07:38:33-00:00",
    "Login": "git_user1",
    "Avatar": ""
  },
  "Committer": {
    "Name": "git_user1",
    "Email": "git_user1@tencent.com",
    "Date": "2012-06-07T07:38:33-00:00",
    "Login": "git_user1",
    "Avatar": ""
  },
  "Link": "https://git.code.tencent.com/xinnjie/testme/commit/6269835cafdd89e6d8b8d4a4e3738c79206c1d06"
}
